pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -61,9 +62,10 @@ type queueAttr struct {
 	ancestors []api.QueueID
 	children  map[api.QueueID]*queueAttr
 
-	deserved  *api.Resource
-	allocated *api.Resource
-	request   *api.Resource
+	deserved                               *api.Resource
+	allocated                              *api.Resource
+	allocatedWithoutSchedulingDisabledNode *api.Resource
```

Comment:
Why should we add a new field rather than just directly modify `allocated`?

Reply:
I'm not sure if it will affect other logic. If ok, I modify allocated